main/Helper.cpp

```diff
@@ -526,50 +526,54 @@ std::vector<std::string> ExecuteCommandAndReturn(const std::string &szCommand, i
 	return ret;
 }
 
-//convert date string 10/12/2014 10:45:58 en  struct tm
-void DateAsciiTotmTime (std::string &sTime , struct tm &tmTime  )
+std::string TimeToString(time_t ltime, _eTimeFormat format)
```

Comment:
Well thinking about it I guess for consistency purposes you're right and we should use a pointer.. Will change that!